packages/lib/pod/src/podUtil.ts

```diff
@@ -230,9 +256,20 @@ export function checkPODValue(
         POD_INT_MAX
       );
       break;
-    case "eddsa-pubkey":
+    case EDDSA_PUBKEY_TYPE_STRING:
       requireValueType(nameForErrorMessages, podValue.value, "string");
-      checkPublicKeyFormat(podValue.value);
+      try {
+        checkPublicKeyFormat(podValue.value);
+      } catch (e) {
+        if (
+          e instanceof TypeError &&
+          e.message === "Public key should be 32 bytes hex-encoded."
```

Comment:
This dependency on a specific error string is definitely going to break when someone changes the string (e.g. I'm planning to change this format to allow Base64 encoding).

You could address that with a targeted unit test, so that a failure would lead me to this line in such a case.

Alternately, does this block really need to exist?  Why not just do the re-throw on any TypeError?  Or why not just pass `nameForErrorMessages` into checkPublicKeyFormat (as an optional arg?) to make the whole catch-and-rethrow unnecessary?